ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```
@@ -2763,6 +2767,43 @@ tests:
     test:
     - chain: openshift-e2e-test-netobserv-qe
     workflow: cucushift-installer-rehearse-vsphere-ipi
+- as: baremetalds-ipi-ovn-dualstack-primaryv6-f28-netobserv
+  capabilities:
+  - intranet
+  cron: 16 22 23 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v6v4
+        MIRROR_IMAGES=true
+        INSTALLER_PROXY=true
+        NETWORK_TYPE=OVNKubernetes
+      TEST_FILTERS: Network_Observability
+      TEST_IMPORTANCE: .*
+      TEST_SCENARIOS: with VMs|53844
+      TEST_TIMEOUT: "55"
+    test:
+    - ref: netobserv-apply-redhat-operators
+    - ref: netobserv-deploy-nfs-provisioner
+    - ref: cucushift-pre
+    - ref: openshift-extended-test
+    - ref: openshift-e2e-test-qe-report
```

we could create 2 chains for these steps since these are repeated across the jobs:
chain 1:

```suggestion
    - ref: netobserv-apply-redhat-operators
    - ref: netobserv-deploy-nfs-provisioner
```
chain 2
```suggestion
    - ref: cucushift-pre
    - ref: openshift-extended-test
    - ref: openshift-e2e-test-qe-report
```

---

chain 1 -> netobserv-prepare-bm-cluster
chain 2 -> replace by openshift-e2e-test-netobserv-qe